### ISSUE TITLE
hg38?

### ISSUE THREAD

**Issue by ahdee** [tommyau/itdseek#5]
I changed the coordinate to:  chr13:28033024-28035453 in the sh file.  Seem to be working?

**Comment by tommyau** [comment]
Yes. If you want to apply ITDseek to a different reference genome build, you need to edit file itdseek.sh last line (line 31): replace chr13:28607161-28609590 (hg19) with chr13:28033024-28035453 (hg38).

**Comment by ahdee** [comment]
@tommyau thanks for confirming! maybe add a parameter check? just a thought. thanks.

**Comment by tommyau** [comment]
Thanks for your suggestion. For backward compatibility, the current genomic positions chr13:28607161-28609590 will still be the default.
Do you think something like this will help?
./itdseek.sh <sample.bam> <ref.fasta> <samtools> chr13:28033024-28035453
